SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialEventTracer.h

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Interop/Connection/SpatialTraceEvent.h"
+#include "SpatialView/EntityComponentId.h"
+
+#include <WorkerSDK/improbable/c_io.h>
+#include <WorkerSDK/improbable/c_trace.h>
+
+// Documentation for event tracing in the GDK can be found here: https://brevi.link/gdk-event-tracing-documentation
+
+DECLARE_LOG_CATEGORY_EXTERN(LogSpatialEventTracer, Log, All);
+
+// TODO - Individual RPC Calls (distinguishing between GDK and USER)
+// TODO - RPCs on newly created objects go through a different flow and need to be handled
+
+namespace SpatialGDK
+{
+// Note: SpatialEventTracer wraps Trace_EventTracer related functionality
```

Comment:
Nit - Would remove the "Note:" and put this on the line above the class definition.